### PR TITLE
[project-base] remove left panel, add horizontal menu

### PR DESCRIPTION
**Pull request by TomasGottvald**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Remove left panel from all pages, add categories from left panel to new horizontal menu.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: project-base/templates/Front/Content/Category/panel.html.twig [hunk 1]
```diff
@@ -1,12 +1,15 @@
 {% if categoriesWithLazyLoadedVisibleChildren|length > 0 %}
-    <ul class="js-category-list list-menu dont-print {% if isFirstLevel %}list-menu--root{% endif %}" {% if isFirstLevel %}id="js-categories"{% endif %}>
+
+    <ul class="js-category-list list-menu dont-print {% if isFirstLevel %}list-menu--root{% endif %}
+    {% if categoriesWithLazyLoadedVisibleChildren[0].category.level > 2 %}list-menu--dropdown{% endif %}
+    " {% if isFirstLevel %}id="js-categories"{% endif %}>
```
Comment (TomasLudvik) [inline project-base/templates/Front/Content/Category/panel.html.twig]: Please format this in better way, this is hard to read. Refactor classes to variables etc.

File: project-base/templates/Front/Content/ContactForm/contactForm.html.twig [hunk 2]
```diff
@@ -1,35 +1,40 @@
...
+                    }
+                )}}
+            {% endif %}
+            <div class="text-right">
+                {{ form_widget(form.send, { label: 'Send'|trans }) }}
+            </div>
 
-{{ form_end(form) }}
-{{ init_js_validation(form, true) }}
+        {{ form_end(form) }}
```
Comment (TomasLudvik) [inline project-base/templates/Front/Content/ContactForm/contactForm.html.twig]: form_start is at beginning of file so end should be at the end of file ?

File: project-base/templates/Front/Content/ContactForm/contactForm.html.twig [hunk 3]
```diff
@@ -1,35 +1,40 @@
...
-                label:  'I agree with privacy policy.'|trans,
-                errors_attr: { class: 'form-error--choice' },
-                rowClass: 'form-line-block'
-            }
-        )}}
-    {% endif %}
-    <div class="text-right">
-        {{ form_widget(form.send, { label: 'Send'|trans }) }}
-    </div>
+        {{ form_errors(form) }}
```
Comment (TomasLudvik) [inline project-base/templates/Front/Content/ContactForm/contactForm.html.twig]: weird indentation

File: project-base/templates/Front/Content/ContactForm/index.html.twig [hunk 4]
```diff
@@ -1,46 +1,50 @@
...
+                            <span class="js-validation-errors-list js-validation-error-list-{{ form.privacyPolicy.vars.id }} form-error form-error--choice display-none">
+                                <span class="form-error__icon">
+                                    <i class="svg svg-warning"></i>
+                                </span>
+                                <ul class="form-error__list"></ul>
+                            </span>
+                        </div>
+                    </dd>
+                </dl>
+                {{ form_widget(form.send, { label: 'Send'|trans }) }}
```
Comment (TomasLudvik) [inline project-base/templates/Front/Content/ContactForm/index.html.twig]: Bad indentation

File: upgrade/UPGRADE-v9.0.0-dev.md [hunk 5]
```diff
@@ -200,6 +200,153 @@ There you can find links to upgrade notes for other versions too.
 
 - vats can be create and manage per domains ([#1498](https://github.com/shopsys/shopsys/pull/1498))
     - please read [upgrade instruction for vats per domain](https://github.com/shopsys/shopsys/blob/9.0/upgrade/upgrade-instruction-for-vats-per-domain.md)
+
+- remove left panel web layout and add horizontal menu ([#1540](https://github.com/shopsys/shopsys/pull/1540))
+    - if you have your custom design you can skip this all task about twig files
+    - because we don't have left panel on frontend anymore we have to center banner slider (or change its width to 100% - don't forget to change image size and reupload images from administration) [src/Resources/styles/front/common/components/box/slider.less](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Resources/styles/front/common/components/box/slider.less)
```
Comment (TomasLudvik) [inline upgrade/UPGRADE-v9.0.0-dev.md]: It should be enough to change sizes

File: project-base/templates/Front/Content/ContactForm/contactForm.html.twig (hunk 2 above)
Comment (TomasGottvald) [reply project-base/templates/Front/Content/ContactForm/contactForm.html.twig]: Fixed, start and end of form added into webline

File: project-base/templates/Front/Content/Category/panel.html.twig (hunk 1 above)
Comment (TomasGottvald) [reply project-base/templates/Front/Content/Category/panel.html.twig]: I tried to format i better way. But actually dont know how use variables - I dont think it will be better. Can you draw your idea how to do it?

File: project-base/templates/Front/Content/ContactForm/contactForm.html.twig (hunk 3 above)
Comment (TomasGottvald) [reply project-base/templates/Front/Content/ContactForm/contactForm.html.twig]: Reformated

File: project-base/templates/Front/Content/ContactForm/index.html.twig (hunk 4 above)
Comment (TomasGottvald) [reply project-base/templates/Front/Content/ContactForm/index.html.twig]: Reformated

File: upgrade/UPGRADE-v9.0.0-dev.md (hunk 5 above)
Comment (TomasGottvald) [reply upgrade/UPGRADE-v9.0.0-dev.md]: Reupload deleted

File: project-base/src/Resources/styles/front/common/components/box/slider.less [hunk 6]
```diff
@@ -10,8 +11,10 @@
         margin-bottom: @margin-gap;
         width: @box-slider-width;
         max-width: 100%;
+        margin: 0 auto @box-slider-bottom-gap auto;
         visibility: hidden;
 
+
```
Comment (Srnka392) [inline project-base/src/Resources/styles/front/common/components/box/slider.less]: Remove empty line

**Comment by TomasGottvald** [comment]
@malyMiso Thx for edge cases, I made small update, now you can have as many items as you want. It will separate items to rows, and submenu opens under menu - no matter how many rows it has. Can you test it please?
